patcher/Patcher.py

```
@@ -324,51 +288,37 @@ def align(v, a):
-    folders = scan_for_headers_in_section(section_folder_path)
-    includes = " ".join((f"-I ../section/{folder}/" for folder in folders))
+        for path in list_files_at(section_folder_path, "**/*.cxx", ["main.cxx"]):
+            main_file.writelines(f"#include \"{path}\"\n")
 
     if run_system(
-            f"""cd {build_folder_path} &
-            {clang_compiler_path} {CLANG_FLAGS}
-            -I ../include/ {includes}
-            ../section/main.cxx -o clangfile.o"""):
+            t"""{config.clang_path} -c {" ".join(config.clang_flags)} -I {include_folder_path} {section_folder_path / "main.cxx"} -o {build_folder_path / "clangfile.o"}"""):
         raise Exception("Errors occurred during building of cxx files")
```

Ruff (0.14.14)

Create your own exception

(TRY002)